lib/mnesia/src/mnesia_loader.erl

```diff
@@ -744,7 +744,17 @@ do_send_table(Pid, Tab, Storage, RemoteS) ->
 	    Storage ->
 		%% Send first
 		TabSize = mnesia:table_info(Tab, size),
-		KeysPerTransfer = calc_nokeys(Storage, Tab),
+		KeysPerTransfer =
+                case ?catch_val(send_table_batch_size) of
+                    {'EXIT', _} ->
+                        mnesia_lib:set(send_table_batch_size, 0),
```

Comment:
this will crash because if `?catch_val(send_table_batch_size)` returns `EXIT`,
it means the ets table does not exist.